Automattic/jetpack
# AI Extension: Avoid restarting Form insertion

Fixes #

## Proposed changes:

* Update `setContent` to look for current blocks and not replace everything.

### Other information:


## Jetpack product discussion


N/A

## Does this pull request change what data or activity we track or use?


N/A

## Testing instructions:






* Insert `Form` in editor.
* Ask for an initial form.
* Keep updating the form.
* Check if it doesn't start from beginning and it keeps working.

#### Demo

https://github.com/Automattic/jetpack/assets/1663717/f4b9002e-7222-43c8-b450-65113359dbce
